functions/postSurveyComplete.protected.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+
+export interface Event {
+  Channel: string;
+  CurrentTask: string;
+  Memory: string;
+  UserIdentifier: string;
+}
+
+type EnvVars = {
+  FLEX_PROXY_SERVICE_SID: string;
+};
+
+const deleteProxySession = async (context: Context<EnvVars>, proxySession: string) => {
+  try {
+    const client = context.getTwilioClient();
+    const ps = await client.proxy
+      .services(context.FLEX_PROXY_SERVICE_SID)
```

Comment:
Even though `staging` and `production` are not using `beta-action`, this function would be deployed, correct?
Is there any chance this function would be called on `staging` or `production`? (because if so, `context.FLEX_PROXY_SERVICE_SID` would be undefined)

Reply:
Yes it will be deployed but no, it should never be invoked. The breaking change is in the task router callback tho.